coss/base/static/css/coss.scss

```diff
@@ -0,0 +1,124 @@
+@import 'colors';
```

Comment:
Shouldn't this file along with others that are referring to the content be part of the `home` app and not the `base`?